process: Added issue templates- Code-Health, Research, Process (#352)
google/android-fhir: .github/ISSUE_TEMPLATE/research.md

---
name:  Research

> about: Issues related to creation and managing of research tasks. Includes background, goal, and methodology.

Reviewer: ```suggestion
about: Research tasks
```

> 

Reviewer: add a row
```
labels: research
```

---

**Describe the Issue**

> <!--Please state here specifics of what you want to do.-->

Reviewer: ```
**Describe the issue to be researched**
Include any background information and available resources.

**Describe the goal of the research**
What's the desired outcome of this task? What artifacts should be produced?

**Describe the methodology**
What information is available for the research? Where can more information be found? Who should the assignee approach to ask questions?
```

---
name:  Research
about: Research tasks
title: ''
labels: 'research'
assignees: ''

---

**Describe the issue to be researched**
Include any background information and available resources.

**Describe the goal of the research**
What's the desired outcome of this task? What artifacts should be produced?

**Describe the methodology**

> What information is available for the research? Where can more information be found? Who should the assignee approach to ask questions?

Reviewer: tweak this a little:

```suggestion
Where can more information be found? Who should the assignee approach to ask questions? How can a decision be made?
```

thanks!
